.github/workflows/iroha2-profiling-image.yml

```
@@ -47,20 +41,20 @@ jobs:
         uses: docker/setup-buildx-action@v3
         with:
           install: true
-      - name: Build and push iroha2:profiling-image
+      - name: Build and push IROHA2_PROFILE image
         uses: docker/build-push-action@v5
         if: always()
         with:
           push: true
           tags: |
-            hyperledger/iroha2:${{ github.event.inputs.IROHA2_IMAGE_TAG }}-${{ github.event.inputs.IROHA2_IMAGE_RELEASE }}-profiling
-            docker.soramitsu.co.jp/iroha2/iroha2:${{ github.event.inputs.IROHA2_IMAGE_TAG }}-${{ github.event.inputs.IROHA2_IMAGE_RELEASE }}-profiling
+            hyperledger/iroha:${{ env.TAG }}-IROHA2_PROFILE
+            docker.soramitsu.co.jp/iroha2/iroha:${{ env.TAG }}-IROHA2_PROFILE
```

```suggestion
            hyperledger/iroha:${{ env.TAG }}-$IROHA2_PROFILE
            docker.soramitsu.co.jp/iroha2/iroha:${{ env.TAG }}-$IROHA2_PROFILE
```

I'm not sure if you need `$` here